stage1/units/units/halt.target

```diff
@@ -1,4 +0,0 @@
-[Unit]
-Description=Halt
-DefaultDependencies=no
-AllowIsolate=yes
```

Comment:
To add a little bit more context: the `halt.target` has a `Requires/After` dependency on `shutdown.service` in the case of mutable pods. This prevents mutable pods to shut down prematurely if there are no applications started.